strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/repository/proxied/LocalStorageProxyRepositoryExpiredArtifactsCleaner.java

```diff
@@ -113,11 +114,12 @@ private void deleteFromStorage(final List<ArtifactEntry> artifactEntries)
     {
         for (final ArtifactEntry artifactEntry : artifactEntries)
         {
-            final Storage storage = configurationManager.getConfiguration().getStorage(artifactEntry.getStorageId());
+            final Storage storage = configurationManager.getConfiguration().getStorage(
+                    artifactEntry.getStorageId());
```

Comment:
Cosmetic: Could be on the previous line.

Reply:
done